cypress/e2e/unauthUserCanAccessSimulation.cy.js

```diff
@@ -2,36 +2,23 @@
 /* SPDX-License-Identifier: MIT-0 */
 
 describe("Unauth Simulation", () => {
-	beforeEach(() => {
-		cy.visitDomain(`${Cypress.env("WEB_DOMAIN")}/demo`);
+	context("Desktop view", () => {
+		beforeEach(() => {
+			cy.visitDomain(`${Cypress.env("WEB_DOMAIN")}/demo`);
+		});
+	
+		it("US-001 - should allow user to use unauth simulation", () => {
+			cy.useUnauthSimulation(false)
```

Comment:
Can this be changed to take in an enum? This currently reads like "don't use unauth simulation", not "use unauth simulation, but not with responsive view". An enum would make this more readable.
Ex: `cy.useUnauthSimulation(DesktopView)`

This applies to all of the functions taking in a boolean right now to switch between DesktopView and ResponsiveView.

Reply:
resolved